src/libs/actions/Report.js

```diff
@@ -323,6 +352,76 @@ function fetchChatReportsByIDs(chatList) {
         });
 }
 
+/**
+ * Update stored IOU object for given chat report
+ *
+ * @param {Object} report
+ */
+function updateIOUReportDataForChatReport(report) {
+    updateIOUReportData(report).then((iouReportObject) => {
+        let chatReportObject = {};
+        if (iouReportObject) {
+            const iouReportKey = `${ONYXKEYS.COLLECTION.REPORT_IOUS}${iouReportObject.reportID}`;
+            Onyx.merge(iouReportKey, iouReportObject);
+            chatReportObject = {
+                iouReportID: iouReportObject.reportID,
+                hasOutstandingIOU: iouReportObject.stateNum === 1
+                    && iouReportObject.total !== 0,
+            };
+        } else {
+            chatReportObject = {
+                hasOutstandingIOU: false,
+            };
+        }
+        const reportKey = `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`;
+        Onyx.merge(reportKey, chatReportObject);
+    });
+}
+
+/**
+ * Update stored chat report object for given IOU report
+ *
+ * @param {Object} report
+ */
+function updateChatReportDataForIOUReport(report) {
```

Comment:
These IOU<>CHAT report functions can be a bit confusing. Perhaps naming the param `chatReport` might slightly help here.

Reply:
Renamed it as `iouReport`.